api/routes_account.go

```diff
@@ -77,6 +77,23 @@ func (api *API) registerAirDrop(c *gin.Context) {
 		api.LogError(err, err.Error())(c, http.StatusBadRequest)
 		return
 	}
+	mqURL := api.cfg.RabbitMQ.URL
+	qm, err := queue.Initialize(queue.EmailSendQueue, mqURL, true, false)
+	if err != nil {
+		api.LogError(err, eh.QueueInitializationError)(c, http.StatusBadRequest)
+		return
+	}
+	es := queue.EmailSend{
+		Subject:     "airdrop registered",
+		Content:     fmt.Sprintf("user %s with airdrop code %s to eth address %s is registered", username, aidropID, ethAddress),
+		ContentType: "text/html",
+		UserNames:   []string{"admin"},
+		Emails:      []string{"admin@rtradetechnologies.com"},
```

Comment:
not sure how I feel about this hardcoded - probs best to have this be configurable via env, otherwise anyone who tries to spin up their own Temporal will be shooting emails at us haha

Reply:
though there'll be a few instances of this email address that needs to be replaced I think

Reply:
Good idea, i'll create a jira ticket to cleanup hard-coded emails